File: patch_schema.py
import sys

with open("prisma/schema.prisma", "r") as f:
    content = f.read()

search = """  julesChatForwardDelay Int      @default(5) // In minutes
  createdAt             DateTime @default(now())"""

replace = """  julesChatForwardDelay Int      @default(5) // In minutes
  aiSystemPrompt        String? // Top-level prompt for AI bots
  commentTemplate       String? // Markdown template for individual bot comments
  createdAt             DateTime @default(now())"""

content = content.replace(search, replace)

with open("prisma/schema.prisma", "w") as f:
    f.write(content)

File: patch_schema.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'prisma', 'schema.prisma');
let content = fs.readFileSync(filePath, 'utf8');

// Insert groupName into Repository model
content = content.replace("createdAt              DateTime            @default(now())", "groupName              String?             @default(\"Default\")\n  createdAt              DateTime            @default(now())");

fs.writeFileSync(filePath, content);
console.log("Schema updated");
